tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -375,6 +374,16 @@ def test_get_replays_user_filters(self):
                 "device.brand:Apple",
                 "device.family:Macintosh",
                 "device.model:10",
+                # Contains operator.
+                f"id:[{replay1_id},{uuid.uuid4().hex},{uuid.uuid4().hex}]",
+                # Or expression.
+                f"id:{replay1_id} OR id:{uuid.uuid4().hex} OR id:{uuid.uuid4().hex}",
+                # Paren wrapped expression.
+                f"((id:{replay1_id} OR id:b) AND (duration:>15 OR id:d))",
+                # Implicit paren wrapped expression.
+                f"(id:{replay1_id} OR id:b) AND (duration:>15 OR id:d)",
+                # Implicit And.
+                f"(id:{replay1_id} OR id:b) OR (duration:>15 platform:javascript)",
```

Comment:
do these tests validate what is / isn't in the response? could we add a test that ensures e.g. and AND clause removes a row, while an OR clause keeps a row with a condition the row doesn't satisfy?

Reply:
👍